setup/teardown_tests.sh

```diff
@@ -2,8 +2,8 @@
 
 echo "Removing environment from "${CONDA_TEMP_PREFIX}
 source deactivate ${CONDA_TEMP_PREFIX}
-conda env remove --prefix ${CONDA_TEMP_PREFIX}
+conda env remove --yes --prefix ${CONDA_TEMP_PREFIX} 
 unset CONDA_TEMP_PREFIX
 rm conf/net/int_service/giles_conf.json
-rm conf/net/ext_service/habitica_conf.json
-rm conf/net/ext_service/push_conf.json
+rm conf/net/ext_service/habitica.json
+rm conf/net/ext_service/push.json
```

Comment:
also, why do you explicitly remove these?  the build always performs a 'git clean -fdx' before re-cloning the repo.

Reply:
no strong reason - just for symmetry with the setup, where I create those files.